packages/turf-meta/index.js

```diff
@@ -32,64 +32,73 @@ function coordEach(geojson, callback, excludeWrapCoord) {
         wrapShrink = 0,
         currentIndex = 0,
         isGeometryCollection,
-        isFeatureCollection = geojson.type === 'FeatureCollection',
-        isFeature = geojson.type === 'Feature',
+        type = (geojson) ? geojson.type : null,
```

Comment:
@DenisCarriere isn't geojson required here? I mean, shouldn't this actually throw an error if `!geojson`?
~~Then `type = geojson.type` any time, as `geojson.type` can be `null`~~
Sorry, `type` can NOT be `null`, it would be an invalid GeoJSON.